### PR TITLE
refactor(subscriberdb): Reformatting subscriberdb code from changes to 5G auth vectors

### PR DESCRIPTION
**Pull request by ardzoht**
Signed-off-by: Alex Rodriguez <ardzoht@gmail.com>

## Summary

- Updating some leftover formatting changes from #8051 

## Test Plan

- make test on magma VM

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Review by themarwhal — APPROVED** [review]
thank you!
have to make the check mandatory soon :/

**Review by ulaskozat — APPROVED** [review]
thanks for taking care of this.

File: lte/gateway/python/magma/subscriberdb/crypto/milenage.py [hunk 1]
```diff
@@ -12,12 +12,12 @@
...
 import hmac
+
 from Crypto.Cipher import AES
 from Crypto.Random import random
-from .lte import (
-    BaseLTEAuthAlgo,
-    FiveGRanAuthVector,
-)
+
+from .lte import BaseLTEAuthAlgo, FiveGRanAuthVector
```
Comment (ulaskozat) [inline lte/gateway/python/magma/subscriberdb/crypto/milenage.py]: change to `magma.subscriberdb.crypto.lte`

**Comment by ulaskozat** [comment]
failing lte_test?

File: lte/gateway/python/magma/subscriberdb/protocols/m5g_auth_servicer.py [hunk 2]
```diff
@@ -14,12 +14,17 @@
 import logging
 
 from google.protobuf.json_format import MessageToJson
+from lte.protos.subscriberauth_pb2 import M5GAuthenticationInformationAnswer
```
Comment (Lkishor123) [inline lte/gateway/python/magma/subscriberdb/protocols/m5g_auth_servicer.py]: It passes the lte test if we use from (lte.protos import subscriberauth_pb2_grpc, subscriberauth_pb2).

**Comment by ulaskozat** [comment]
complains about commit message. interestingly insync-checkin has failed as well. rerunning.

**Review by pshelar — APPROVED** [review]

**Pull request merged**